### ISSUE TITLE
docs-validation is failing

### ISSUE THREAD

**Issue by kolyshkin** [cri-o/cri-o#4424]
make docs-generation && ./hack/tree_status.sh is currently failing on circleci:

```
bin/crio -d "" --config="" md  > docs/crio.8.md
INFO[0000] Starting CRI-O, version: 1.20.0-dev, git: de186def35ffb7e67f5bfb3ebcd11addd97a0c41(clean) 
bin/crio -d "" --config="" man > docs/crio.8
INFO[0000] Starting CRI-O, version: 1.20.0-dev, git: de186def35ffb7e67f5bfb3ebcd11addd97a0c41(clean) 
tree is dirty, please commit all changes

 M docs/crio.8.md
diff --git a/docs/crio.8.md b/docs/crio.8.md
index 04ff8a383..57c9b5aec 100644
--- a/docs/crio.8.md
+++ b/docs/crio.8.md
@@ -276,15 +276,15 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--seccomp-use-default-when-empty**="": Use the default seccomp profile when an empty one is specified (default: false)
 
-**--selinux**: Enable selinux support (default: true)
+**--selinux**: Enable selinux support (default: false)
 
 **--separate-pull-cgroup**="": [EXPERIMENTAL] Pull in new cgroup (default: "")
 
 **--signature-policy**="": Path to signature policy JSON file. (default: "", to use the system-wide default)
 
-**--storage-driver, -s**="": OCI storage driver (default: "overlay")
+**--storage-driver, -s**="": OCI storage driver (default: "")
 
-**--storage-opt**="": OCI storage driver option (default: [overlay.mountopt=nodev])
--More--
```

The selinux is probably because of missing `libselinux1-dev` package.

**Comment by saschagrunert** [comment]
Do you plan to propose a fix PR before we merge https://github.com/cri-o/cri-o/pull/4423, or what do you suggest?

**Comment by kolyshkin** [comment]
> Do you plan to propose a fix PR before we merge #4423, or what do you suggest?

Yes, I guess this needs to be fixed before we merge #4423. Looking...

**Comment by haircommander** [comment]
it passes now!
